index.html

```diff
@@ -5,9 +5,173 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="/node_modules/normalize.css/normalize.css">
     <link rel="stylesheet" href="styles/styles.css">
-    <title>Base Project</title>
+    <link href="https://fonts.googleapis.com/css?family=Anaheim|Berkshire+Swash&display=swap" rel="stylesheet">
+    <title>Restaurant Website</title>
 </head>
 <body>
-    
+
+    <section class="header">
```

Comment:
Instead of using a <section> tag here, consider using a <header> tag is it will be more semantically correct